docs/paper/paper.bib

```diff
@@ -170,3 +170,31 @@ @article{emig_prl_2006
   year    = {2006},
   doi     = {10.1103/PhysRevLett.96.080403}
 }
+
+% +-------------+
+% | Experiments |
+% +-------------+
+
+% https://doi.org/10.1103/PhysRevB.100.081406
+@article{liu_prb_2019,
+  author  = {Liu, M. and Xu, J. and Klimchitskaya, G. L. and Mostepanenko, V. M. and Mohideen, U.},
+  title   = {{Examining the Casimir puzzle with an upgraded AFM-based technique and advanced surface cleaning}},
+  journal = {Physical Review B},
+  volume  = {100},
+  issue   = {8},
+  pages   = {081406},
```

Comment:
081406(R)

Reply:
To be honest, I was not surce whether to put the (R) or not. Therefore, I checked the bibtex citation export of PRA and there was no (R). I have added it (
8eb84453075611625e89d081c2680b7e215b552f).

Reply:
It is written like that in the headline of the paper, that's why I proposed it.